apps/aehyperchains/src/connectors/aehc_chain_sim_connector.erl

```diff
@@ -2,23 +2,80 @@
 
 -behaviour(aehc_connector).
 
--export([broadcast/3, verify/1, subscribe/1]).
+-behaviour(gen_server).
 
--spec broadcast(_From, _TxPayload, _To) ->
-                       binary().
-broadcast(From, TxPayload, To) ->
+%% API.
+
+-export([start_link/0]).
+
+%% gen_server.
+-export([init/1]).
+-export([handle_call/3]).
+-export([handle_cast/2]).
+-export([handle_info/2]).
+-export([terminate/2]).
+
+-export([send_tx/1, get_block/1]).
+
+%% API.
+
+id() -> 
+    ?MODULE.
+
+-spec start_link() ->
+                        {ok, pid()} | ingnore | {error, term()}.
+start_link() ->
+    Id = id(),
+    gen_server:start_link({local, Id}, ?MODULE, [], []).
+
+%%%===================================================================
+%%%  aehc_connector behaviour
+%%%===================================================================
+
+-spec send_tx(Tx::aetx:tx()) -> binary().
+send_tx(Tx) ->
+    gen_server:call(id(), {send_tx, Tx}).
+
+-spec get_block(Num::integer()) -> aehc_connector:block().
+get_block(Num) ->
+    gen_server:call(id(), {get_block, Num}).
+
+%%%===================================================================
+%%%  gen_server behaviour
+%%%===================================================================
+
+-record(state, { pid::pid() }).
+
+init([]) ->
+    %% TODO TO ask opt from config;
+    Opts = #{},
+    {ok, Pid} = aec_chain_sim:start(Opts),
+    io:fwrite("Parent chain's connector ~p is attached: ~p", [?MODULE, Pid]),
+    {ok, #state{ pid=Pid }}.
```

Comment:
What's up with that weird formatting?
![image](https://user-images.githubusercontent.com/1352848/91422975-30b29f00-e858-11ea-90d5-8aad99569751.png)

Make spaces around =